src/conn/conn_capacity.c

```diff
@@ -306,6 +306,7 @@ __wt_capacity_throttle(WT_SESSION_IMPL *session, uint64_t bytes,
 
 	conn = S2C(session);
 	cap = &conn->capacity;
+	/* NOLINTNEXTLINE(clang-analyzer-deadcode.DeadStores) */
```

Comment:
Clang Scan doesn't like that we're assigning capacity to 0 and then reassigning it immediately in the switch statement below. I tried removing the assignment but my compiler complains that we might be using capacity when it's uninitialised.

It seems that we can't win so I've just suppressed the linter here.